README.md

```diff
@@ -1,31 +1,31 @@
-# unc0ver
+# Unc0ver
 ### The most advanced jailbreak tool
 ![unc0ver logo](https://raw.githubusercontent.com/pwn20wndstuff/Undecimus/master/Undecimus/Assets.xcassets/AppIcon.appiconset/Icon-App-60x60%403x.png?token=AlyO4xDujoguob2DCFfUbNI8jO82OyCgks5bx5ZPwA%3D%3D)
```

Comment:
Can we try to fix all of these weird links?

Reply:
Then I will have to Re-upload uncover logo

Reply:
I don't think so, we can just change it to `![unc0ver logo](/Undecimus/Assets.xcassets/AppIcon.appiconset/Icon-App-60x60%403x.png)`